deployer/debug.py

```diff
@@ -259,5 +260,167 @@ def start_docker_proxy(
         subprocess.check_call(cmd)
 
 
+@app.command()
+def transfer_old_home_dir_to_new_location(
+    cluster_name: str = typer.Argument(..., help="Name of cluster to operate on"),
+    hub_name: str = typer.Argument(..., help="Name of hub to operate on"),
+    source_user: str = typer.Option(
+        prompt=True,
+        confirmation_prompt=True,
+        help="Name of source user who's home dir files we are transferring",
+    ),
+    destination_user: str = typer.Option(
+        prompt=True,
+        confirmation_prompt=True,
+        help="Name of destination user in who's home dir we will be moving the files",
+    ),
+):
+    """
+    Copy all the files from a user's old home dir to the new home directory
+    """
+    # Name the pod so we know what to delete when the transfer is done
+    pod_name = f"{cluster_name}-{hub_name}-transfer-shell"
+    pod = {
+        "apiVersion": "v1",
+        "kind": "Pod",
+        "spec": {
+            "terminationGracePeriodSeconds": 1,
+            "automountServiceAccountToken": False,
+            # Make sure the pod and containers can only be ran as the jupyter user
+            # Otherwise the files we copy might end up be owned by another user
+            "securityContext": {"runAsUser": 1000, "runAsGroup": 1000},
+            "volumes": [
+                # This PVC is created by basehub
+                {"name": "home", "persistentVolumeClaim": {"claimName": "home-nfs"}}
+            ],
+            "containers": [
+                {
+                    "name": pod_name,
+                    # Use ubuntu image so we get better GNU cp
+                    "image": "ubuntu:jammy",
+                    "stdin": True,
+                    "stdinOnce": True,
+                    "tty": True,
+                    "volumeMounts": [
+                        {
+                            "name": "home",
+                            "mountPath": "/home",
+                        }
+                    ],
+                    "securityContext": {"allowPrivilegeEscalation": False},
+                }
+            ],
+        },
+    }
+
+    # Command that creates the pod described above
+    create_pod_cmd = [
+        "kubectl",
+        "-n",
+        hub_name,
+        "run",
+        pod_name,
+        "--overrides",
+        json.dumps(pod),
+        "--image",
+        # Use ubuntu image so we get GNU cp and other tools
+        # Should match what we have in our pod definition
+        "ubuntu:jammy",
```

Comment:
```suggestion
        "ubuntu:22.04",
```

Reply:
This was specifying `image` directly, but its also part of the pod override. Its nice to only use one so that we don't make mistakes updating only one but not the other in the future.